packages/accounts-controller/package.json

```diff
@@ -76,8 +76,7 @@
     "webextension-polyfill": "^0.12.0"
   },
   "peerDependencies": {
-    "@metamask/keyring-controller": "^19.1.0",
-    "@metamask/network-controller": "^22.2.1",
```

Comment:
We omit this dep (which is inherited from the `multichain-network-controller`).

Reply:
Does this controller talk to `keyring-controller` and `network-controller`? [Peer dependencies aren't inherited](https://dev.to/arcanis/implicit-transitive-peer-dependencies-ed0), so if so then this still needs to be here.

Reply:
I reviewed `accounts-controller` and I think you're right about `network-controller` at least. It looks we are just using this for types and not for the messenger, so I wonder if that really belongs in `dependencies` and not `dev` + `peerDependencies`?

Reply:
Sorry again, I jumped the gun on this, I just see you are downgrading `keyring-controller`, so maybe all we need is to move `network-controller` to `dependencies`.

Reply:
Since it's just for types, we could leave it in the `devDependencies` only, no?

Reply:
Types are included in the build for type declaration, so we've moved this to the dependencies

Reply:
We had to also add it to the peer dependencies because `yarn constraints` couldn't reference the package. @mcmire mentioned that this is a bug that will need to be addressed